website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,412 @@
+
+Delete any usages of <code>[application:didRegisterUserNotificationSettings:](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623022-application?language=objc)</code> and <code>RCTPushNotificationManager</code>’s corresponding <code>didRegisterUserNotificationSettings:</code> as well.
+
+**Example:** See the RNTester <code>[AppDelegate.mm](https://github.com/facebook/react-native/blob/main/packages/rn-tester/RNTester/AppDelegate.mm)</code>.
+
+#### JS
+
+1. Remove any references to `alertAction`.
+2. Remove the `handler` argument on any calls to `removeEventListener`.
+3. Replace any usages of `repeatInterval` by firing multiple notifications using `fireDate` or `fireIntervalSeconds` instead.
+4. Note that `soundName` will be null when it is accessed on a `Notification` returned from `getScheduledLocalNotifications()` and `getDeliveredNotifications()`.
+
+</details>
+
+### Android SDK minimum bump (Android 6.0)
+
+React Native 0.74 has a minimum SDK version requirement of 23 (Android 6.0). See our justification for this change [here](https://github.com/react-native-community/discussions-and-proposals/discussions/740).
+
+### Yarn 3 for New Projects
+
+[Yarn 3](https://yarnpkg.com/blog/release/3.0) is now the default JavaScript package manager for new projects initialized with React Native Community CLI.
+
+Yarn 3.x will be used with `nodeLinker: node-modules`, a mode providing compatibility with React Native libraries. This replaces Yarn Classic (1.x, deprecated) as the previous default. To upgrade Yarn version inside your existing app you can follow this [guide](https://yarnpkg.com/migration/guide).
+
+```sh
+$ yarn --help
+━━━ Yarn Package Manager - 3.6.4 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  $ yarn <command>
+```
+
+The Community CLI also supports initializing projects with other package managers via the `--pm` flag ([read more](https://github.com/react-native-community/cli/blob/main/docs/init.md)).
+
+## Breaking Changes
+
+These are some of the key breaking changes in 0.74. Please consult the [full changelog](https://github.com/facebook/react-native/blob/main/CHANGELOG.md) for the complete list of breaking changes.
+
+- Remove web performance logging from `GlobalPerformanceLogger` ([#41299](https://github.com/facebook/react-native/pull/41299)).
+- Make `start/end` in styles always refer to writing direction ([#42251](https://github.com/facebook/react-native/pull/42251)).
+- **Android**: Get rid of old APIs in `FabricUIManagerProvider` ([#42059](https://github.com/facebook/react-native/pull/42059)).
+- **iOS**: Delete `configFilename` and `configKey` arguments from iOS codegen CLI ([#41533](https://github.com/facebook/react-native/pull/41533)).
+
```

Comment:
We need to add:
- **iOS**: changed how the [`bundleURL`](https://github.com/facebook/react-native/pull/43994) is handled. Before, it was set when React Native was started in an instance variable and it was not possible to update it. Now, it takes a function (usually the [`bundleURL`(https://github.com/facebook/react-native/blob/0.74-stable/packages/react-native/template/ios/HelloWorld/AppDelegate.mm#L22-L29)] from the AppDelegate) so the URL is re-evaluated when needed. This unblock the possibility to update the bundleURL across refreshes. This should affect your app only if you were changing the `bundleURL` variable after the app is started. In this case, move the logic that updates the variable to the `bundleURL` function in the `AppDelegate`.